src/java/com/sarality/app/datastore/db/TableQueryExecutor.java

```diff
@@ -0,0 +1,101 @@
+package com.sarality.app.datastore.db;
+
+import com.sarality.app.datastore.Column;
+import com.sarality.app.datastore.query.BaseQueryBuilder;
+import com.sarality.app.datastore.query.FilterValue;
+import com.sarality.app.datastore.query.Query;
+import com.sarality.app.datastore.query.QueryGenerator;
+
+import java.util.List;
+
+/**
+ * Builds and executes a Query on a Table.
+ *
+ * @author abhideep@ (Abhidep Singh)
+ */
+public class TableQueryExecutor<T> extends BaseQueryBuilder<TableQueryExecutor<T>> {
```

Comment:
I think the locations are different thats why it shows up as different. The old location used to introduce strange directory dependencies. Now the Table, TableQueryBuilder and TableQueryExecutor are all in the same directory. Same thing for DataStore